Source/WTF/wtf/PlatformHave.h

```diff
@@ -1389,7 +1389,7 @@
 #endif
 #endif
 
-#if (PLATFORM(IOS) || PLATFORM(WATCHOS) || PLATFORM(APPLETV))
```

Comment:
So we don't want to enable this for Watch or TV anymore?

Reply:
Yes, I would like to do more testing before enabling.

Thanks for reviewing!